typescript/writer.go

```diff
@@ -9,6 +9,8 @@ type CodeWriter interface {
 	End()
 	Eol()
 	Write(code string)
+	OpenBlock()
```

Comment:
Do we need to expose these? I thought there was a block/inline block already

Reply:
Ill looks again. Perhaps I missed how to do this.

Reply:
because `Begin()` in the typescript [writer](https://github.com/Zocdoc/gopoetry/blob/8e4afb323f954990205fd39fbf70bd13b0aef253/typescript/writer.go#L25) starts with a `Eol()` call, I don't see how there would be a way to change the bracket style to be inline. Perhaps I'm missing something tho?

Reply:
however, I was able to reuse the existing BlockDeclaration.

Reply:
there are some `*Inline` helpers for scala code tho.